### ISSUE TITLE
[QUESTION] Is the stack trace of a crash recorded?

### ISSUE THREAD

**Issue by auermich93** [tingsu/Stoat#21]
I did some testing on a simple app (com.zola.bmi) and Stoat reports a bug event sequence in `stoat_fsm_output/crashes/1/`, but the corresponding `emulator-5554_logcat.txt` file shows the following:

```
--------- beginning of system
--------- beginning of main
10-24 22:34:55.335  1673  1700 E ActivityManager: applyOptionsLocked: Unknown animationType=0
```

I would have expect a complete crash stack trace. Is this a false positive?

**Comment by tingsu** [comment]
Yes, according to the log you provided, it is a false postive. A real app crash should include the app package name at least. 

We will check all the stack traces reported by Stoat to exclude false positives (the rule is discussed in Stoat's paper, see Section 5.4, "Method of Calculating Unique Crashes")
